Installation/cmake/modules/CGAL_SetupCGALDependencies.cmake

```diff
@@ -136,7 +136,7 @@ function(CGAL_setup_CGAL_dependencies target)
     endif()
     if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 3)
       message( STATUS "Using gcc version 4 or later. Adding -frounding-math" )
-      target_compile_options(${target} ${keyword} "-frounding-math")
+      target_compile_options(${target} ${keyword} "$<$<COMPILE_LANGUAGE:CXX>:-frounding-math>")
```

Comment:
Unfortunately, [`COMPILE_LANGUAGE` was introduced in CMake version 3.3][changelog], and CGAL is supposed to support CMake version 3.1 and later.

  [changelog]: https://cmake.org/cmake/help/v3.3/release/3.3.html#generator-expressions

I think there should be two different lines, with a conditional on `CMAKE_VERSION`:
```cmake
if(CMAKE_VERSION VERSION_LESS 3.3)
  target_compile_options(${target} ${keyword} "-frounding-math")
else
  target_compile_options(${target} ${keyword} "$<$<COMPILE_LANGUAGE:CXX>:-frounding-math>")
endif()
```